### PR TITLE
Standardise exports

### PR DESCRIPTION
**Pull request by akademy**
Hello.

Just a tweak and standardising the exports. We had a problem importing the VisitInput into one of our projects.

Pretty basic changes.

### CODE REVIEW

File: src/components/VisitInput.tsx
Comment (VictoriaBeilsten-Edmands) [inline src/components/VisitInput.tsx:1]: Need to update VisitInput import in `VisitInput.stories.tsx` to `import { VisitInput } from "./VisitInput";`

File: src/components/VisitInput.tsx [hunk 1]
```diff
@@ -89,6 +89,5 @@ const VisitInput: React.FC<VisitInputProps> = ({
   );
 };
 
-export default VisitInput;
```
Comment (VictoriaBeilsten-Edmands) [inline src/components/VisitInput.tsx:92]: Need to update VisitInput import in VisitInput.stories.tsx to import { VisitInput } from "./VisitInput";

Comment (akademy) [reply src/components/VisitInput.tsx:92]: Good spot! Have updated. Ta.

Comment (VictoriaBeilsten-Edmands) [reply src/components/VisitInput.tsx:92]: Oh, it's in `VisitInput.test.tsx` too.

**Pull request merged**